dd-sdk-android/apiSurface

```diff
@@ -109,6 +110,8 @@ data class com.datadog.android.core.configuration.Credentials
   constructor(String, String, String, String?, String? = null)
   companion object 
     const val NO_VARIANT: String
+object com.datadog.android.core.configuration.private
+const val VALID_IP_REGEX: String
```

Comment:
strange, this doesn't look right

Reply:
Yup...I noticed that too, I will open a task to investigate this issue with the `apiSurface` generator.